wiki/Katu/en.md

```diff
@@ -9,6 +8,56 @@ tags:
 
 *See also: [Geki](/wiki/Geki)*
 
-**Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note. This idea originates from the Nintendo DS game Elite Beat Agents which is one of the games the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+*Not to be confused with osu!taiko's Katu*
 
-<!-- TODO: Add links -->
+**Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note.
+However, this can't be achieved by having a 50 or lower in the combo itself.
+
+There are two kind of Katu, which is based on the combo finishing judgment. A 300-ended Katu rewards more health bonus than 100-ended Katu, but always worse than a Geki.
+
+This idea originates from the Nintendo DS game Elite Beat Agents which is one of the games the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+
+## Screenshots
+
+In here provided the screenshots of obtaining a Katu/Katsu/Beat from Elite Beat Agents, Ouendan 2, and osu! itself.
+
+![](img/eba-without-100k.jpg) ![](img/oto-countdwn-300k.jpg)
+
+![](img/osu-cutepnth-100k.jpg)
+
+![](img/osu-stardisc-300k.jpg)
+
+## Example gameplay links
+
+- [Elite Beat Agents: Without a Fight/Jumpin' Jack Flash - YouTube](https://www.youtube.com/watch?v=wPOCmzY_fqs)
+- [Osu! Tatakae! Ouendan 2: Countdown - YouTube](https://www.youtube.com/watch?v=6us1tY8jOSI)
+
+## Other game modes
+
+### osu!taiko
+
+Katu is the judgment given when you hit big notes at same time. osu!taiko have more leniency on handling both health bonus and score bonus for hitting both keys within short timespan.
+
+### osu!catch
+
+Katu is counted for every droplets the player failed to catch. This value is not shown in result screen.
+
+### osu!mania
+
+On osu!mania, Katu is replaced with **200**. As the name states it gives 200 worth of score with slight penalty on the scoring side.
+
+## Other game aspects
+
+### Storyboard (DS Games)
+
+Katu triggers the second-tier on the storyboard during the gameplay. Usually shown with natural spirit shown during the sequence.
+
+### Storyboard (osu!)
+
+Storyboard in osu! mostly intended for osu!standard. Obtaining Katu make several events occurs:
+
+- Fail Layer activates.
+- Pass Layer deactivates.
+- Triggers "Failing" handler if transitioning from Pass state.
+
+To note that Fail side of storyboard in osu! is different than DS Games.
```

Comment:
this is vague and gives no specific description, plus it is missing in the Geki article, mind removing it?

Reply:
this gives a clear difference from DS game actually. Geki in standard works the same as how it is on the DS.
Katu meanwhile is rather something in osu!'s implementation.

To explain more that on DS there are **3 layers** of succession rate.
- Bad (obtained if 50 or lower is in respective combo), shows how tasks are improperly done.
- Good (obtained if 100 or lower is in respective combo), on m980's authentic mapset, this side is shown on Fail side. Due to osu! SB trigger only checks 300 only combo or have some non-300 in it. (This assumes standard mode)
- Great (obtained if only 300 is in respective combo), not on this article for sure.

Reply:
I see, but:

1) this is outside of the scope of these articles, and
2) this should go to the Storyboarding article or the likes

which is why you should exclude this specific sentence from this article